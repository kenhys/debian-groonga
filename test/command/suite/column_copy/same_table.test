table_create Logs TABLE_NO_KEY
column_create Logs timestamp_text COLUMN_SCALAR Text

load --table Logs
[
{"timestamp_text": "2015-08-12 00:00:00"},
{"timestamp_text": "2015-08-12 00:00:01"},
{"timestamp_text": "2015-08-12 00:00:02"}
]

column_create Logs timestamp_time COLUMN_SCALAR Time
column_copy Logs timestamp_text Logs timestamp_time

dump --dump_schema no
