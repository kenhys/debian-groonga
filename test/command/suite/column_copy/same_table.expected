table_create Logs TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Logs timestamp_text COLUMN_SCALAR Text
[[0,0.0,0.0],true]
load --table Logs
[
{"timestamp_text": "2015-08-12 00:00:00"},
{"timestamp_text": "2015-08-12 00:00:01"},
{"timestamp_text": "2015-08-12 00:00:02"}
]
[[0,0.0,0.0],3]
column_create Logs timestamp_time COLUMN_SCALAR Time
[[0,0.0,0.0],true]
column_copy Logs timestamp_text Logs timestamp_time
[[0,0.0,0.0],true]
dump --dump_schema no
load --table Logs
[
["_id","timestamp_text","timestamp_time"],
[1,"2015-08-12 00:00:00",1439305200.0],
[2,"2015-08-12 00:00:01",1439305201.0],
[3,"2015-08-12 00:00:02",1439305202.0]
]
